src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -28,8 +28,8 @@ private Storage() {
         storageMotor.burnFlash();
     }
 
-    public void run() {
-        storageMotor.set(StorageConstants.motorSpeed);
```

Comment:
I'll leave this up to @matunkar as the controls lead, but either this needs put back and storage always runs at a constant speed OR the input speed needs to be clamped to a min and max value before being set

Reply:
Thinking about how storage usually works, I believe this should be put back. Storage usually runs forward or backward at a constant speed in order to feed into acquisition consistently